### PR TITLE
Gh9010 yahoo options parsing bug

### LINKED ISSUE #9010: Yahoo options DataReader can't parse underlying prices with commas
I ran into this problem retrieving option data for the SPX settlement index:

```
>>> options_object = web.Options('^spxpm', 'yahoo')
>>> options_object.get_call_data( expiry=datetime.date(2014, 12, 20) )
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/pollyp/anaconda/lib/python2.7/site-packages/pandas/io/data.py", line 785, in get_call_data
    return self._get_data_in_date_range(expiry, call=True, put=False)
  File "/Users/pollyp/anaconda/lib/python2.7/site-packages/pandas/io/data.py", line 1098, in _get_data_in_date_range
    frame = self._get_option_data(expiry=expiry_date, name=name)
  File "/Users/pollyp/anaconda/lib/python2.7/site-packages/pandas/io/data.py", line 717, in _get_option_data
    frames = self._get_option_frames_from_yahoo(expiry)
  File "/Users/pollyp/anaconda/lib/python2.7/site-packages/pandas/io/data.py", line 655, in _get_option_frames_from_yahoo
    option_frames = self._option_frames_from_url(url)
  File "/Users/pollyp/anaconda/lib/python2.7/site-packages/pandas/io/data.py", line 684, in _option_frames_from_url
    self.underlying_price, self.quote_time = self._get_underlying_price(url)
  File "/Users/pollyp/anaconda/lib/python2.7/site-packages/pandas/io/data.py", line 696, in _get_underlying_price
    .getchildren()[0].text)
ValueError: invalid literal for float(): 2,071.92
```

Here's my show_versions output:
```
>>> pd.show_versions()

INSTALLED VERSIONS
------------------
commit: None
python: 2.7.8.final.0
python-bits: 64
OS: Darwin
OS-release: 13.4.0
machine: x86_64
processor: i386
byteorder: little
LC_ALL: None
LANG: None

pandas: 0.15.1
nose: 1.3.4
Cython: 0.21
numpy: 1.9.0
scipy: 0.14.0
statsmodels: 0.5.0
IPython: 2.2.0
sphinx: 1.2.3
patsy: 0.3.0
dateutil: 2.2
pytz: 2014.7
bottleneck: None
tables: 3.1.1
numexpr: 2.3.1
matplotlib: 1.4.0
openpyxl: 1.8.5
xlrd: 0.9.3
xlwt: 0.7.5
xlsxwriter: 0.5.7
lxml: 3.4.0
bs4: 4.3.2
html5lib: 0.9999-dev
httplib2: None
apiclient: None
rpy2: None
sqlalchemy: 0.9.7
pymysql: None
psycopg2: None
```

### PR DESCRIPTION
**Pull request by PollyP**
closes #9010. Fix for Yahoo Options parse error for underlying prices of 1,000.00 or larger.

### CODE REVIEW

File: pandas/io/data.py [hunk 1]
```diff
@@ -698,8 +698,19 @@ def _option_frames_from_url(self, url):
 
     def _get_underlying_price(self, url):
         root = self._parse_url(url)
-        underlying_price = float(root.xpath('.//*[@class="time_rtq_ticker Fz-30 Fw-b"]')[0]\
-            .getchildren()[0].text)
+
+	underlying_price = root.xpath('.//*[@class="time_rtq_ticker Fz-30 Fw-b"]')[0]\
```
Comment (jreback) [inline pandas/io/data.py]: looks like this needs 1 more space to format properly

**Comment by jreback** [comment]
@PollyP 

pls address the above, rebase and squash. ping when green.

**Comment by PollyP** [comment]
OK, will do this this weekend.

**Comment by PollyP** [comment]
I wound up syncing and reapplying my changes to create a new PR #9198. I believe it is ready to go.

**Comment by jreback** [comment]
replaced by #9198

**Pull request closed without merging**